content/devices/perseus.md

```diff
@@ -59,7 +59,7 @@ Flash recovery (using Computer)
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
+Congratulations, if everything went well you should be booting into Droidian.
```

Comment:
The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes should reflect here.